package.json

```diff
@@ -28,6 +28,7 @@
     "dev:pidcat": "pidcat org.celo.mobile.dev -t ReactNativeJs -t ReactNativeJS",
     "dev:emulator:list": "$ANDROID_HOME/emulator/emulator -list-avds",
     "dev:emulator:run": "$ANDROID_HOME/emulator/emulator -avd",
+    "depcheck": "depcheck",
```

Comment:
This is actually useful -- without it depcheck flags depcheck as an unused devDependency